integration/meta-typings/google/protobuf/wrappers.ts

```diff
@@ -655,134 +655,134 @@ export const protoMetadata: ProtoMetadata = {
           path: [4, 0],
           span: [50, 0, 53, 1],
           leadingComments:
-            ' Wrapper message for `double`.\n\n The JSON representation for `DoubleValue` is JSON number.\n',
+            ' Wrapper message for `double`.\r\n\r\n The JSON representation for `DoubleValue` is JSON number.\r\n',
```

Comment:
@Larkooo FWIW these changes (and the `*.bin` file changes) might be from running `proto2bin:local` instead of the docker-based `proto2bin`? 

We used to have a lot of churn on these files from contributors having different versions of `protoc` installed on their machines, but in theory the docker-based setup that @boukeversteegh put in place _should_ have removed this churn.

Can you try the docker-based commands and see if it resolves these differences? Or were you already doing that, and we still have some non-determinism issues?